src/panels/lovelace/entity-rows/hui-input-number-entity-row.ts

```diff
@@ -132,7 +132,6 @@ class HuiInputNumberEntityRow extends LitElement implements LovelaceRow {
     return css`
       :host {
         display: block;
-        cursor: pointer;
```

Comment:
In slider mode the slider itself ensures we have a pointer. In text mode, it should not be a pointer, but a text edit cursor instead (which is anyway the browser default for inputs).